index.html

```diff
@@ -0,0 +1,148 @@
+            <li class="advantages-item-first advantages-item">
+              <h3 class="advantages-title-first">Настоящий городок</h3>
+              <p class="advantages-description-first">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+              <img class="advantages-photo-first" src="images/photo-1.jpg" width="800" height="385" alt="Фото города.">
+            </li>
+            <li class="advantages-item-second advantages-item">
+              <h3 class="advantages-title-second">Небольшая площадь</h3>
+              <p class="advantages-description-second">Все достопримечательности находятся очень близко</p>
+            </li>
+            <li class="advantages-item-third advantages-item">
+              <h3 class="advantages-title-third">Красивая дорога</h3>
+              <p class="advantages-description-third">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+            </li>
+            <li class="advantages-item-fourth advantages-item">
+              <h3 class="advantages-title-fourth">Мало туристов</h3>
+              <p class="advantages-description-fourth">Большинство едет в Гранд Каньон и толпится там</p>
+            </li>
+            <li class="advantages-item-fifth advantages-item">
+              <img class="advantages-photo-second" src="images/photo-2.jpg" width="800" height="385" alt="Фото каньона.">
+              <h3 class="advantages-title-fifth">Там есть мост дьявола</h3>
+              <p class="advantages-description-fifth">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+            </li>
+          </ul>
+      </section>
+      <section class="advantages-service">
+        <h2>Приезжайте в Седону отдохнуть в комфорте и уюте!</h2>
+        <p>Опытный персонал и качественное обслуживание!</p>
+        <ul class="advantages-service-list">
+          <li class="advantages-service-item-first advantages-service-item">
+            <h3 class="advantages-housing">Жильё</h3>
+            <p class="advantages-description">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+          </li>
+          <li class="advantages-service-item-second advantages-service-item">
+            <h3 class="advantages-food">Еда</h3>
+            <p class="advantages-description">Всегда заказывайте топовый фирменный бургер, вы не разочаруетесь!</p>
+          </li>
+          <li class="advantages-service-item-third advantages-service-item">
+            <h3 class="advantages-souvenirs">Сувениры</h3>
+            <p class="advantages-description">Не только китайского, но и настоящего местного производства!</p>
+          </li>
+        </ul>
+      </section>
+      <section class="reservations-search">
+        <h2 class="reservations">Заинтересовались?</h2>
+        <p class="reservations-text">Укажите предполагаемые даты поездки и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <a class="reservations-button" href="#">Поиск гостиницы в Седоне</a>
+      </section>
```

Comment:
```suggestion
      <section class="reservations">
        <h2 class="reservations-title">Заинтересовались?</h2>
        <p class="reservations-text">Укажите предполагаемые даты поездки и мы покажем вам лучшие предложения гостиниц в Седоне</p>
        <a class="reservations-button" href="#">Поиск гостиницы в Седоне</a>
      </section>
```

И соответственно в стилях